src/patterns/telephone-numbers/index.md.njk

```diff
@@ -22,7 +22,7 @@ Only collect telephone numbers from people if you genuinely need them. Not every
 ## How it works
 
 ### Allow different formats
-Users should be allowed to enter telephone numbers in whatever format is familiar to them. You should allow for additional spaces, hyphens, brackets and dashes, and be able to accommodate country and area codes.
+Let users enter telephone numbers in whatever format is familiar to them. Allow for additional spaces, hyphens, dashes and brackets, and be able to accommodate country and area codes.
```

Comment:
Makes wording consistent with similar content on the [bank details](https://design-system.service.gov.uk/patterns/bank-details/#allow-different-formats-for-the-sort-code) and [payment card details](https://design-system.service.gov.uk/patterns/payment-card-details/#allow-different-formats) patterns.